### PR TITLE
support eventID separator as a command-line option 

### PR DESCRIPTION
**Pull request by drinkingkazu**
a minor change so that a user doesn't have to locally install larnd-sim nor modify the system path install to use a string value "eventID" as a separator (the default kept as spillID)

### CODE REVIEW

**Comment by drinkingkazu** [comment]
Added another piece of change that logs the time and memory usage over simulation steps for understanding the performance in laymans term w/o using nvprof outside.

**Comment by krwood** [comment]
#111 reads in `EVENT_SEPARATOR` and other formerly hard coded configurations through a configuration file instead of from the command line. I would like to merge that in and then revise this PR to get your memory logger in. How does that sound to you?

**Comment by drinkingkazu** [comment]
@krwood config file sounds good. yes please merge the mem logger! sorry i was traveling till April...

**Comment by drinkingkazu** [comment]
Closing to open a new PR for memory logger
